src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -111,6 +114,9 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                     for initFile in initFiles:
                         copyCommand += "source %s\n" % initFile
 
+        if args.wma_disablewriterecovery and not args.wma_cerncastor:
```

Comment:
I know I wrote that line, but I think we should remove the check on wma_cerncastor here. It's redundant since the only place where we use castor mode is in LogCollect and we control that code.